nameko/messaging.py

```diff
@@ -389,6 +355,14 @@ def connection(self):
         )
         return Connection(self.amqp_uri, heartbeat=heartbeat)
 
+    def handle_message(self, provider, body, message):
+        ident = "{}.handle_message[{}]".format(
+            type(provider).__name__, message.delivery_info['routing_key']
+        )
+        self.container.spawn_managed_thread(
+            lambda: provider.handle_message(body, message), identifier=ident
```

Comment:
i think elsewhere we do this with a `partial` instead of a `lambda` though not sure about pros and cons. maybe one less function call / shorter call stack?